models/iaf_psc_delta.nestml

```diff
@@ -124,8 +124,8 @@ neuron iaf_psc_delta_neuron:
       # for decay until end of refractory period
       if with_refr_input:
         refr_spikes_buffer += spikes * exp(-r * h / tau_m) * 1 mV
-      else:
-        spikes.get_sum() # clear buffer entry, ignore spike
+      #else:
```

Comment:
With the removal of get_sum(), I am not sure what NESTML construct would be used to clear the buffer. Hence my commenteing out the old, dysfunct code, pending review.